proto/connectrpc/conformance/v1alpha1/client_compat.proto

```diff
@@ -80,16 +70,20 @@ message ClientResponseResult {
   repeated Header response_headers = 1;
   repeated ConformancePayload payloads = 2;
   // The error received from the actual RPC invocation. Note this is not representative
-  // of a runtime error and should always be the proto equivalent of a Connect error
+  // of a runtime error and should always be the proto equivalent of a Connect error.
   Error error = 3;
-  // In case the client cannot decode Any from JSON, it should instead return the received JSON
-  repeated google.protobuf.Struct error_details_raw = 7;
   repeated Header response_trailers = 4;
+  // When processing an error from a Connect server, this should contain
+  // the actual JSON received on the wire. Clients under test do not need
+  // to populate this field. It is primarily used by the reference client,
+  // so that a server-under-test's on-the-wire error can be examined and
+  // validated.
+  google.protobuf.Struct connect_error_raw = 5;
```

Comment:
I think this name is a little more descriptive, since this field only applies to the Connect protocol.